src/app.js

```diff
@@ -68,11 +99,11 @@ io.on('connection', function(socket){
   socket.on('live', function(formData) {
     var req = {body: formData};
     generator.createDistDir(req, socket, function(appFolder, url) {
-      socket.emit('live.ready', {
-        appDir: appFolder,
-        url: url
-      });
-      //generator.showLivePreview(req, socket);
+        folder = appFolder;
```

Comment:
@Princu7 
Why do we need to store a reference in variable called `folder` . Are we going to use it somewhere.